.travis.yml

```diff
@@ -16,7 +16,7 @@ notifications:
 env:
   global:
     - POST_PROCESS='echo "post processing reached"'
-    - BEFORE_SCRIPT='echo current dir: $(pwd)'
+    - "BEFORE_SCRIPT='echo current dir: $(pwd)'"
```

Comment:
Does this need to be documented for the user's doc or not?

Reply:
I don't think so. This is a pretty rare case in which ":<SPACE>" is forcing the line to be intepreted as mapping, at least with pyyaml.